managed-databases/postgresql-and-mysql/how-to/monitor-databases-cockpit.mdx

```diff
@@ -0,0 +1,133 @@
+  - Any Read Replicas
+
+This page explains how to get started with Scaleway Cockpit for viewing your Managed Database metrics and logs.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
+
+## How to activate your Cockpit
+
+Scaleway Cockpit is activated by default if you have created any [Scaleway resources integrated with Cockpit](/observability/cockpit/reference-content/cockpit-limitations/#product-integration-into-cockpit).
+
+## How to access the Grafana dashboard
+
+To view your Managed Database's metrics and logs, use the Grafana dashboard which is accessible from the Scaleway console:
+  - First, you must create a user and their associated credentials (a username and password) for Grafana
+  - Then, you can use these credentials to access the Grafana dashboard for all your Scaleway products, via the Scaleway console.
+
+1. Create a Grafana user and retrieve their username and password by [following this procedure](/observability/cockpit/how-to/retrieve-grafana-credentials/).
+
+2. From the Scaleway console [Cockpit overview page](https://console.scaleway.com/cockpit/overview), click **Open dashboards** in the top right corner.
+
+3. Enter your Grafana username and password to log in.
+
+    You are directed to the Scaleway Cockpit dashboards homepage.
+
+## How to view and understand your Managed Database metrics
+
+1. Click **Open dashboards** on the top right from the Scaleway Cockpit dashboards homepage.
+2. Click either **RDB MySQL Overview** or **RDB PostgreSQL Overview**, depending on your engine of choice, on the left side menu.
+
+    You are directed to the RDB Overview dashboard.
+
+    <Lightbox src="scaleway-grafana-rdb.webp" alt="" />
+
+Metrics are refreshed every minute, and report on the following data:
+
+- **CPU usage**
+- **Memory usage**
+- **Total connections**
+- **Disk usage** (in % and in Bytes)
+- **Replication lag in seconds** (when High Availability mode is activated or when Read Replicas are created).
+
+
+### Choosing metric parameters
+
+At the top of the dashboard, you can configure the following parameters, that control which metrics are displayed:
+
+- **Resource name** - Select a specific Database Instance to monitor.
+- **Node** - Choose a specific node (main, replica, or Read Replica). In an High Availability (HA) cluster, failovers may show the same node as both main and replica. For nodes in an HA cluster, you might see two verions of the same node, the main and the replica. The replica will start appearing as if any failovers happen and will show only metrics for any activity that happened over the failover period.
```

Comment:
```suggestion
- **Node** - Choose a specific node (main, replica, or Read Replica). In a High Availability (HA) cluster, failovers may show the same node as both main and replica. For nodes in an HA cluster, you might see two versions of the same node, the main and the replica. The replica will start appearing as if any failovers happen and will show only metrics for any activity that happened over the failover period.
```

Reply:
This part is not very clear to me. I have another suggestion:
"In an High Availability (HA) cluster, both nodes might switch from main to replica and vice versa. So you might see two versions of the same node, as main and as replica. If you select main (respectively replica) for a specific node, you will only see activity of the node when he was main (resp. replica)"

wdyt?

Reply:
```suggestion
- **Node** - Choose a specific node (main, replica, or Read Replica). In a High Availability (HA) cluster, both nodes might switch from main to replica and vice versa. So you might see two versions of the same node, as main and as replica. If you select main for a specific node, you will only see the node's activity when it was the main, and vice-versa.
```